prow/cluster/boskos.yaml

```diff
@@ -91,3 +91,19 @@ spec:
     protocol: TCP
     port: 80
     targetPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: boskos-http-metrics
+  namespace: test-pods
+spec:
+  selector:
+    app: boskos
+  ports:
+  - name: metrics
+    port: 9090
+    protocol: TCP
+    targetPort: 9090
+  loadBalancerIP: [fill this in]
```

Comment:
Will this dummy value cause problems? (i.e. do we need to emit the field entirely until we have a valid value?)

Reply:
right, I don't intend to submit this until we've reserved an IP address. I can do that, assuming there's consensus that this is a good idea.